integration/v3_grpc_test.go

```diff
@@ -567,6 +565,15 @@ func TestV3WatchMultiple(t *testing.T) {
 // TestV3WatchMultipleEventsFromCurrentRevision tests Watch APIs from current revision
 // in cases it receives multiple events.
 func TestV3WatchMultipleEventsFromCurrentRevision(t *testing.T) {
+	testV3WatchMultipleEvents(t, 0, false)
+}
+
+// TestV3WatchMultipleEventsFromUnsynced tests Watch APIs from unsynced watchers.
+func TestV3WatchMultipleEventsFromUnsynced(t *testing.T) {
+	testV3WatchMultipleEvents(t, 1, true)
+}
+
+func testV3WatchMultipleEvents(t *testing.T, startRev int64, isTxn bool) {
```

Comment:
OK, if txn/put matters, can you add tests for testV3WatchMultipleEvents(t, 0, true) and testV3WatchMultipleEvents(t, 1, false)?